src/main/java/org/traccar/config/Keys.java

```diff
@@ -100,6 +100,13 @@
     public static final ConfigKey FORWARD_JSON = new ConfigKey(
             "forward.json", Boolean.class);
 
+    /**
+     * Boolean value to enable URL parameters in json mode. For example, {uniqueId} for device identifier,
+     * {latitude} and {longitude} for coordinates.
+     */
+    public static final ConfigKey FORWARD_JSON_URL_VARIABLES = new ConfigKey(
+        "forward.json.url.variables", Boolean.class);
```

Comment:
I think we should make this shorter. Something like `forward.urlVariables`.

Reply:
I thought about it, but it will mean that this property enables URL variables for both modes (post, get). Is it ok to rename property to `forward.urlVariables` and change logic with the following
`formattedUrl = urlVariables ? url : formatRequest(position);` ?
Notice: it will change logic for existing users

Reply:
No, it shouldn't change non JSON. Static URL with no payload makes no sense and it will break existing users.

Reply:
Done